src/components/fields/edit/DateEditor.jsx

```diff
@@ -40,43 +33,24 @@ export default function DateEditor(props) {
     <div>
       <MuiPickersUtilsProvider utils={DateFnsUtils}>
         <div style={{ display: 'flex', flexDirection: 'column' }}>
-          <KeyboardDatePicker
+          <KeyboardDateTimePicker
             disableToolbar
             disableFuture
             variant="inline"
             autoOk
-            format="yyyy-MM-dd" // US: MM/dd/yyyy
+            ampm={false} // US: true
+            format="yyyy-MM-dd HH:mm" // US: MM/dd/yyyy hh:mm
```

Comment:
What modulates whether this get's the U.S. treatment vs. not? Here, it looks hard-coded to _not_ be U.S. format.

Reply:
for MVP all times and dates are hardcoded to be international. but later this will be a user preference